logging/logger_manager.go

```diff
@@ -41,3 +52,40 @@ func (lr *loggerRegistry) updateLoggerLevel(name string, level Level) error {
 	logger.SetLevel(level)
 	return nil
 }
+
+func (lr *loggerRegistry) getRegisteredLoggerNames() []string {
+	lr.mu.RLock()
+	defer lr.mu.RUnlock()
+	registeredNames := make([]string, 0, len(loggerManager.loggers))
+	for name := range lr.loggers {
+		registeredNames = append(registeredNames, name)
+	}
+	return registeredNames
+}
+
+// Exported Functions specifically for use on global logger manager.
+
+// RegisterLogger registers a new logger with a given name.
+func RegisterLogger(name string, logger Logger) {
+	loggerManager.registerLogger(name, logger)
+}
+
+// DeRegisterLogger attempts to remove a logger from the registry and returns a boolean denoting whether it succeeded.
+func DeRegisterLogger(name string) bool {
+	return loggerManager.deregisterLogger(name)
+}
+
```

Comment:
per [this comment](https://github.com/viamrobotics/rdk/pull/4204#issuecomment-2236675867), do we still need these?

```suggestion
```

Reply:
`RegisterLogger` is necessary, `DeRegisterLogger` is helpful for tests. I don't think there is a way to include a test in `reader_test.go` without this `DeRegisterLogger`.

Reply:
ah sorry, missed that usage!